mimesis/builtins/pl.py

```diff
@@ -90,9 +94,10 @@ def regon(self) -> str:
 
         :return: valid 9-digit REGON
         """
-        args = (8, 9, 2, 3, 4, 5, 6, 7)
+        regon_coefficients = (8, 9, 2, 3, 4, 5, 6, 7)
```

Comment:
Typo here: `region_`

Reply:
It's not a typo: [REGON](https://pl.wikipedia.org/wiki/REGON).